app/src/main/java/fr/free/nrw/commons/nearby/Place.java

```diff
@@ -26,12 +26,14 @@
     private final String category;
     public final String pic;
     public final String destroyed;
```

Comment:
Regarding to this comment of @nicolas-raoul , which I agree https://github.com/commons-app/apps-android-commons/issues/4260#issuecomment-783781535  I think we should remove destroyed and end time properties from Place.java and keep only exists variable instead.

Reply:
Sorry about that, I have made the changes now.